week-2-wide-perspective/w02e02-data-manipulation/package.json

```diff
@@ -1,5 +1,22 @@
 {
   "name": "w02e02-data-manipulation",
   "version": "1.0.0",
-  "dependencies": {}
+  "scripts": {
+    "start": "vite",
+    "build": "vite build"
```

Comment:
To polecenie powinno być tutaj @kdrzymalski :)
np.
```json
"start:backend": "json-server --watch db.json"
```

Właśnie między innymi po to są `"scripts"`